src/brogue/Movement.c

```diff
@@ -2166,6 +2166,25 @@ boolean search(short searchStrength) {
     return foundSomething;
 }
 
+void manualSearch() {
+    short i, j, x, y;
+
+    // Guaranteed discovery of all adjacent locations
+    for (i = -1; i < 2; i++) {
+        x = player.xLoc + i;
+        for (j = -1; j < 2; j++) {
+            y = player.yLoc + j;
+            if (coordinatesAreInMap(x, y)) {
```

Comment:
I think we ought to only discover cells if they are visible by the player

Reply:
Right, I guess adjacent squares can be invisible because of magical darkness or a cursed RoC

Reply:
Hmm, yeah I had forgotten that adjacent tiles can pretty much never be invisible. Still, good safeguard